src/code/z_parameter.c

```diff
@@ -338,8 +290,11 @@ void func_8010EBA0(s16 timer, s16 timerId);
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80112C0C.s")
 
-s16 sAmmoRefillCounts[] = {
-    5, 10, 20, 30, 10, 30, 40, 50, 20, 10, 1, 5, 1, 5, 10, 20, 50, 100, 200, 0,
+s16 sAmmoRefillCounts[] = { 5, 10, 20, 30 };
```

Comment:
Most of it makes way more sense, but what's this one? Nuts? Bombs?

Reply:
It's used for sticks, nuts, and bombs. Since it wasn't one in particular, I just left it as general

Reply:
Ah, better add a comment then.